tests/themes-project/themes/project_theme/theme.ini

```diff
@@ -0,0 +1,12 @@
+name = "Projects theme"
+license = "MIT"
+licenselink = ""
+description = "And Example project theme for testing"
+homepage = ""
+tags = ["simple", "minimal", "projects"]
+features = ["blog"]
+lektor_required_version = 3.1
+
+[author]
```

Comment:
This indentation style doesn't follow the style that Lektor currently has for `.lektorproject` and other `.ini` files as far as I can see.

Reply:
I remove the indentation, (although indentation doesn't matter in ini files)